vaadin-markdown-flow-parent/vaadin-markdown-flow-integration-tests/pom.xml

```diff
@@ -0,0 +1,166 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.vaadin</groupId>
+        <artifactId>vaadin-markdown-flow-parent</artifactId>
+        <version>24.8-SNAPSHOT</version>
+    </parent>
+    <artifactId>vaadin-markdown-integration-tests</artifactId>
+    <packaging>war</packaging>
+    <name>Vaadin Markdown Integration Tests</name>
+    <description>Vaadin Markdown Integration Tests</description>
+    <dependencies>
```

Comment:
A number of dependencies are not relevant:
- flow-data
- flow-lit-template
- flow-polymer-template
- flow-test-util
- vaadin-icons-flow

Reply:
Removed